### DISCUSSION TITLE
Help me to Install

### DISCUSSION THREAD

**Discussion by CiroAntony** [Matt-FTW/dotfiles#9] [Q&A]
Hi dude, I really like your configuration and I'd like to set it up on my system, but I couldn't find any specific guide for installation. In the SETUP section of the README, I found this text:

**  Setup**

** Automatic Installation (Arch Linux)**

Warning
This installation is specifically designed for Arch Linux. If you're running other systems, install the packages with your OS's package manager.

WIP  (Work in Progress)

** Manual Installation (Universal)**

Warning
This installation assumes you have the .config directory in your home directory. Also, note that the names of the packages are from the AUR and Arch Repos. Adapt them to your sistema (system in Spanish).

I'm currently using EndeavourOS, which is based on Arch Linux.

**Comment by Matt-FTW** [comment]
Hey :wave: 
First of all, sorry to answer a little late but I was recently ill and couldnt check my inbox. 

About your question, it says that the installation is still in WIP. Right now I dont have the time nor the energy to create the setup guide. You can try looking at the ansible config files and see if they work for you (havent tested them yet...). 

If you dont want to install it and use it, you could paste the conf files like any other dots (be sure to backup your own files before overwriting them!) and install the dependencies required. The dependencies are located as a list in some of the ansible modules. Here's the URL: https://github.com/Matt-FTW/dotfiles/tree/main/.config/ansible/roles/arch/packages/tasks/modules

If you want only the WM, install some of the packages located in the base.yml file, and so on...

Sorry but right now I cannot help you otherwise. I'll try to have a guide soon. I'll let you know once I have it.

**Comment by Matt-FTW** [comment]
Hi Antony, I uploaded recently a guide on how to install the configuration. Tell me if you have any problem or doubts about it!

**Reply by recluzexyz** [reply]
Bro when I Install your dotfiles , i got no applications inside nothing just wallpapers changing, I'm new to hyprland so any advice

**Reply by Matt-FTW** [reply]
You have to follow the [installation guide](https://dotfiles-docs.vercel.app/getting-started/installation.html). There you have the apps you have to install for the dotfiles to work. I don't cover everything, just the essentials.
